### ISSUE TITLE
saveReport shows only 5 sample row

### ISSUE THREAD

**Issue by 1DanielG** [capitalone/dataCompareR#50]
Is it possible to increase the actual different row that the **saveReport** comes up with? instead of current 5-row preview for each variable? Can a user expand the list when knit the report?

 ex:

```
test <- rCompare(df1, df2 ,keys = 'id' )
saveReport(test, reportName = 'test' , n = 20)
```

*n – The first n different rows

**Comment by rnobleeddy** [comment]
Thanks - it looks like the option to pass this parameter in to `saveReport` is missing. 

It should be trivial to fix - I'll post an update once I've made the changes.

**Comment by rnobleeddy** [comment]
Actually, it looks like this is a special case of #45 - as part of working on this, I'll update both issues, and even if I can't close both of them, I'll consolidate them.

**Comment by rnobleeddy** [comment]
Fix is await PR review. Closing this as a duplicate of #45, only because I wrote all the comments in #45.

@1DanielG The fix is available in the `saveReport-fix` branch right now. We'll merge to master and release a new version in the next few weeks.

**Comment by SaravananSrithar** [comment]
@rnobleeddy Hi, Do we have the solution for increasing the sample size to from 5 to all issues ? Please share the solution. I am looking for it

**Comment by rnobleeddy** [comment]
The following code shows how to get > 5 mismatches to display, and has a workaround to show all.

```
library(dataCompareR)

iris2 <- iris
iris2$Petal.Length <- iris2$Petal.Length + 1
iris2$Sepal.Length[1:10] <- iris2$Sepal.Length[1:10] -1

comp <- rCompare(iris, iris2)

# To choose a specific number of mismatches, set mismatchCount 
summary(comp, mismatchCount = 100)

# There's no built in option for all yet, but this should achieve the same result, by finding the max number of mismatches in the comparison object
summary(comp, mismatchCount = max(unlist(lapply(comp$mismatches, nrow))))

# This argument can be passed to saveReport too
saveReport(comp,   reportName = "test", reportLocation = "~", mismatchCount = 100)
```

More generally, you can access all the mismatches in the `mismatches` item in the object produced by `rCompare`. The mismatches are stored as a list of `data.frames`.

**Comment by SaravananSrithar** [comment]
Hi Rob,

Thanks for the info. Can you tell me how to implement these code or the
code to display all mismatch in the R studio ?

Do I need to completely copy paste the code that you gave ? Or us the
parameters?

I'm looking for options to see the complete mismatches.

Cheers,
Ravan

**Comment by SaravananSrithar** [comment]
Hi Rob, I went ahead and updated as below to print all the mismatches in the dataset. I just updated the printAll only in my savereport function call as below. Will this be sufficinet?

saveReport(comp_dataset, reportName = paste('comp_dataset_', test_case_name,'_',vtimestamp, sep=""),reportLocation = comp_report_location,HTMLReport= TRUE, 
               showInViewer = TRUE, stylesheet = NA, printAll = TRUE)

But, I am getting the error message "Error in saveReport(comp_dataset, reportName = paste("comp_dataset_",  :  unused argument (printAll = TRUE)"
